samples/official/camera-tagging/README.md

```diff
@@ -0,0 +1,116 @@
+# CameraTaggingModule
+
+## Overview
+
+This repository contains the modules necessary to allow individuals to capture images from an RTSP stream, tag them, and upload them to train a Custom Vision model or simply to a Blob store.
+
+It reads 9 environment variables from which 3 are used to build the default URI to the RTSP stream (i.e. rtsp://<RTSP_IP>:<RTSP_PORT>/<RTSP_PATH>), and 4 are used to describe the local blob store. These environment variables can be specified in the docker deployment config.
```

Comment:
What do you mean by docker deployment config?

Reply:
Changed to deployment.template.json.